stellar-captive-core-api/README.md

```diff
@@ -0,0 +1,11 @@
+# Stellar Captive Core API
+The Captive Stellar-Core Server allows you to run a dedicated Stellar-Core instance for the purpose of ingestion. The server must be bundled with a Stellar Core binary.
+
+If you run Horizon with Captive Stellar-Core ingestion enabled Horizon will spawn a Stellar-Core subprocess. Horizon's ingestion system will then stream ledgers from the subprocess via a filesystem pipe. The disadvantage of running both Horizon and the Stellar-Core subprocess on the same machine is it requires detailed per-process monitoring to be able to attribute potential issues (like memory leaks) to a specific service.
```

Comment:
> If you run Horizon with Captive Stellar-Core ingestion enabled Horizon will spawn a Stellar-Core subprocess

I think it can be confusing to start with this info, I'd just say what this package does and skip technical background, at least in the first paragraph.